README.md

```diff
@@ -39,6 +39,14 @@ Getting started
 
 You obviously want to bind to 0.0.0.0 and port 80 when running in <i>production</i>.
 
+## Testing
+
+In order to run the tests and receive a test coverage report, we recommend running `pytest`:
+
+
+    sudo pytest --cov-report term-missing --cov=snare tests/
```

Comment:
```suggestion
    pip install pytest pytest-cov
    sudo pytest --cov-report term-missing --cov=snare snare/tests/
```

Reply:
Let's add some information on how to install dependencies and the complete test path assuming run from the root folder

Reply:
Much Better! Thanks